### PR TITLE
Dev og react bootstrap implemented

### PR DESCRIPTION
**Pull request by omiearicent**

### CODE REVIEW

File: src/components/products/productCarouselView/productsCarousel.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.carousel{
+    position: relative;
+    height: 30%;
+    .carousel-inner{
+    height: 100%;
+    }
+    .carousel-item :active{
+        color: aqua;
+        height: 10px;
```
Comment (sgupta-aricent) [inline src/components/products/productCarouselView/productsCarousel.scss]: remove it

File: src/components/products/productTile/productTile.scss [hunk 2]
```diff
@@ -1,18 +1,29 @@
...
-    img{
-        border: 1px solid #ccc;
+.card{
+    background-color: #f0f2fd;
+   width: 15rem;
+   flex-direction: row-reverse;
+   margin: 1% 0 0% 1%;
+   box-shadow: 3px 3px 5px #aaaaaa;
+   .card-img-top{
+    margin-top: 10px;
```
Comment (sgupta-aricent) [inline src/components/products/productTile/productTile.scss]: use shorthand for margin property

File: src/components/products/productTile/productTile.scss [hunk 3]
```diff
@@ -1,18 +1,29 @@
...
+    margin-right: 10px;
+    width: 79px;
```
Comment (sgupta-aricent) [inline src/components/products/productTile/productTile.scss]: check if relative width can be given if not use same units either px or rem

File: src/components/products/productTile/productTile.scss [hunk 4]
```diff
@@ -1,18 +1,29 @@
...
+    margin-bottom: 10px;
+    background-color: black;
     }
+  }
+  .card:hover{
+    box-shadow: 20px 20px 20px  #aaaaaa;
+  }
+  .card-body {
+    padding: 0.50rem;
+    .card-title {
```
Comment (sgupta-aricent) [inline src/components/products/productTile/productTile.scss:18]: check if it need to be mentioned

File: src/constants/constants.js [hunk 5]
```diff
@@ -1,4 +1,7 @@
 export const PAGE_RANGE = 2;
-export const ITEMS_PER_PAGE = 12;
+export const ITEMS_PER_PAGE = 15;
 export const INITIAL_PAGE = 1;
+export const PRODUCTS_URL = '/res/products.json';
+export const CAROUSEL_PRODUCTS_URL = '/res/carouselProducts.json';
+
```
Comment (sgupta-aricent) [inline src/constants/constants.js:6]: check if it can be optimized

File: src/container/productDashBoard/productDashBoard.scss [hunk 6]
```diff
@@ -5,46 +5,55 @@
...
   .product-tile-container{
-    flex: 1 1 auto;
     display: flex;
     flex-flow: row wrap;
-   // overflow: scroll;
-    margin: auto;
+    flex: 1 0 auto;
+    height: calc(100% - 150px);
+    overflow: auto;
+    overflow: auto;
```
Comment (sgupta-aricent) [inline src/container/productDashBoard/productDashBoard.scss]: remove line 18

**Review by sgupta-aricent — CHANGES_REQUESTED** [review]
Minor changes required

File: src/components/products/productCarouselView/productsCarousel.scss (hunk 1 above)
Comment (omiearicent) [reply src/components/products/productCarouselView/productsCarousel.scss]: removed.

File: src/components/products/productTile/productTile.scss (hunk 2 above)
Comment (omiearicent) [reply src/components/products/productTile/productTile.scss]: done.

File: src/components/products/productTile/productTile.scss (hunk 3 above)
Comment (omiearicent) [reply src/components/products/productTile/productTile.scss]: done.

File: src/components/products/productTile/productTile.scss (hunk 4 above)
Comment (omiearicent) [reply src/components/products/productTile/productTile.scss:18]: Yes it is needed.

File: src/constants/constants.js (hunk 5 above)
Comment (omiearicent) [reply src/constants/constants.js:6]: Will check and try to implement in the upcoming PRs till then requesting you to mark this resolved.

File: src/container/productDashBoard/productDashBoard.scss (hunk 6 above)
Comment (omiearicent) [reply src/container/productDashBoard/productDashBoard.scss]: removed.

**Review by sgupta-aricent — APPROVED** [review]
changes look good

**Pull request merged**